Templates/RemoteRepo/Template/repo.json

```
@@ -9,5 +9,6 @@
     "last_updated": "",
     "gems_data": [],
     "projects_data": [],
-    "templates_data":[]
+    "templates_data":[],
+    "repos":[]
```

nitpick: missing space before `[]`.
```suggestion
    "templates_data": [],
    "repos": []
```